scripts/linters/pylint_extensions.py

```diff
@@ -2064,9 +2240,12 @@ def _process_module_tokens(self, tokens, node):
 
                 if re.search(r'(\s*type:\s*ignore\[)', line):
 
-                    error_codes = re.search(
+                    error_matches = re.search(
                         r'(\s*type:\s*ignore)\[([a-z-\s\,]*)\]', line
-                    ).group(2)
+                    )
+                    error_codes = (
+                        error_matches.group(2) if error_matches else ''
+                    )
```

Comment:
Should we throw an exception if `error_matches` is Falsy since that's what would have happened before?

Reply:
I have edited the regex check in such a way that error_matches won't be falsy.

/cc @sahiljoster32